### ISSUE TITLE
Some accessibility issues in the GUIs

### ISSUE THREAD

**Issue by Kazaf123** [avjinder/Minimal-Todo#171]
**Description:**
This issue aims to figure out some accessibility issues in the following GUI, and we appreciate if such issues could be fixed, which would bring great assistance for visually impaired users.
 
![image](https://user-images.githubusercontent.com/44084361/160950673-376ccfbb-63b7-401f-ae1a-aae92758c648.png)

**Accessibility Issues:**
The textviews of “Settings ” and “About” lack the content-descriptions, resulting in the TalkBack and VoiceOver could not read, also, these texts suffer from low color contrast.
**Appreciate:**
Thanks for reading this bug report and I really hope you will repair these issues for people using this App conveniently.
